rubrics/tool/src/main/java/org/sakaiproject/rubrics/controller/MainController.java

```diff
@@ -38,9 +44,14 @@ public String indexRedirect() {
     @GetMapping("/index")
     public String index(ModelMap model) {
         String token = rubricsService.generateJsonWebToken("sakai.rubrics");
+        String exportPdf = rubricsService.exportPdf();
```

Comment:
Use booleans and a more appropiate name.

boolean enablePdfExport = rubricsService.isPdfExportEnabled();

Reply:
Done.